templates/frontend/objects/article_details.tpl

```diff
@@ -335,7 +335,7 @@
 						</h2>
 						<ul class="value">
 							{foreach from=array_reverse($article->getPublishedPublications()) item=iPublication}
-								{capture assign="name"}{translate key="submission.versionIdentity" datePublished=$iPublication->getData('datePublished')|date_format:$dateFormatShort version=$iPublication->getData('version')}{/capture}
+								{capture assign="name"}{translate key="submission.versionIdentity" datePublished=$iPublication->getData('datePublished')|date_format:$dateFormatShort version=$iPublication->getData('versionDataDisplay')}{/capture}
```

Comment:
@defstat, are you sure this works? I didn't see `setData('versionDataDisplay, ...)` happening anywhere (and would prefer we didn't augment the `DataObject`s with extra attributes beyond what's in the schema).

Reply:
`versionDataDisplay` renamed into 'versionString' is a Schema property of the Publication object, and is set there.

See also [this](https://github.com/pkp/pkp-lib/pull/10810#discussion_r2043814615) comment.